src/app/services/fetched-pokemons/fetched-pokemons-entity.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Pokemon } from './../../models/pokemon';
+import { Injectable } from '@angular/core';
+import {
+  EntityCollectionServiceBase,
+  EntityCollectionServiceElementsFactory,
+} from '@ngrx/data';
+
+@Injectable({ providedIn: 'root' })
+export class FetchedPokemonsEntityService extends EntityCollectionServiceBase<
```

Comment:
I recommend not to use entity services, I suggest to use entity stateful instead